hw1/02-palindrome.js

```diff
@@ -1,3 +1,54 @@
+// Define elements from page that get used by these functions
 const elem = document.querySelector('input');
+const result = document.getElementById('result');
 
-elem.addEventListener('input', handleInput);
+// Checks if input is a palindrome by converting to a string and reversing
+const checkPalindrome = function checkPalindrome() {
+  const value = elem.value;
+
+  // First check if input is valid
+  if (value != '') {
+    if (isNaN(value)) {
+      changeStatus('Input is not a number, please try again.', false);
+      retur1;
```

Comment:
Typo here.

I believe the returns on line 37 and 50 are unnecessary now. A function will implicitly return at the end of its body.

Reply:
Ah thanks for catching that!